demosplan/DemosPlanCoreBundle/Repository/StatementRepository.php

```diff
@@ -511,6 +511,25 @@ public function getExternIdsInUse(string $procedureId): array
         return array_combine($externIds, $externIds);
     }
 
+    /**
+     * @param non-empty-string $procedureId
+     *
+     * @return array<non-empty-string, non-empty-string>
+     */
+    public function getInternIdsInUse(string $procedureId): array
+    {
+        $query = $this->getEntityManager()
+            ->createQueryBuilder()
+            ->select('statement.internId')
+            ->from(Statement::class, 'statement')
+            ->where('statement.procedure = :procedureId')
+            ->setParameter('procedureId', $procedureId)
+            ->getQuery();
+        $internIds = array_column($query->getScalarResult(), 'internId');
+
+        return array_combine($internIds, $internIds);
```

Comment:
May the result contain `null`?

Reply:
done in [5b62cbe](https://github.com/demos-europe/demosplan-core/pull/2575/commits/5b62cbe1599777796d3d460b18a8775cb86c82a6)